Client/game_sa/CPlayerPedSA.cpp

```diff
@@ -75,7 +75,7 @@ CPlayerPedSA::CPlayerPedSA(unsigned int nModelIndex)
     // Set default stuff
     m_pData->m_bRenderWeapon = true;
     m_pData->m_Wanted = pLocalWanted;
-    m_pData->m_fSprintEnergy = 1000.0f;
+    m_pData->m_fTimeCanRun = 1000.0f;
```

Comment:
Does `m_bRenderWeapon` uses wrong field too?

Reply:
You changed this field offset in your first fix pr.
Now this code changes another value